README.md

```diff
@@ -24,20 +24,18 @@ With Gradle from repo.spring.io or Maven Central repositories (stable releases o
 ```groovy
 repositories {
     mavenCentral()
-    maven { url "https://repo.spring.io/milestone" }
+
+    // Milestones
+    // maven { url "https://repo.spring.io/milestone" }
 
     // Snapshots
     // maven { url "https://repo.spring.io/snapshot" }
 }
 
 dependencies {
-    //TODO change to the release artifact and comment milestone repo above when GA
```

Comment:
this TODO is here as a reminder for the team to rework the README a bit once 3.4.0 is about to be released, we'll keep it here.